src/NovelRT/Audio/AudioService.cpp

```diff
@@ -4,16 +4,24 @@
 
 namespace NovelRT::Audio {
 
-AudioService::AudioService() {
-  initializeAudio();
-}
-
-bool AudioService::initializeAudio() {
-  return false;
+AudioService::AudioService() : _logger(LoggingService(Utilities::Misc::CONSOLE_LOG_AUDIO)) {
+  auto device = alcOpenDevice(NULL);
```

Comment:
nit: Everything else is slightly lazy

That is, creating the service doesn't necessarily create all the relevant contexts/devices. Instead they are created when first needed.

This avoids issues where the audio service failing to initialize doesn't bring down the game, for example.

Reply:
Ahh so you're saying that the creation should still call an `initialize` function so that even if it does fail, its not killing everything? (The `initialize` function would actually create the contexts/devices as needed when called.)

Reply:
Tbh, not sure what "slightly lazy" actually means in most contexts

Reply:
Most of the other services are using `Utilities::Lazy<T>` so that the bits are created on first access (and so they can be easily recreated if a device is lost, for example).